website/docs/guides/full-text-search.mdx

```diff
@@ -6,12 +6,12 @@ description: Learn about full-text search parameters and capabilities in FerretD
 # Full-text search
 
 import CodeBlock from '@theme/CodeBlock';
-import MyDataSet from '!!raw-loader!./full-text-search/1-insert-text-data.request.js';
-import CreateIndex from '!!raw-loader!./full-text-search/2-create-text-index.request.js';
-import SearchQuery from '!!raw-loader!./full-text-search/3-basic-text-search.request.js';
+import CreateIndex from '!!raw-loader!./full-text-search/1-create-text-index.request.js';
+import SearchQuery from '!!raw-loader!./full-text-search/2-basic-text-search.request.js';
```

Comment:
Let's use names that match full file names, and group them by file name, not type:

```ts
import BasicTextSearchRequest from '!!raw-loader!./full-text-search/2-basic-text-search.request.js';
import BasicTextSearchResponse from '!!raw-loader!./full-text-search/2-basic-text-search.response.js';
```